src/AtmosphereModels/compute_hydrostatic_pressure.jl

```diff
@@ -14,16 +14,16 @@ const f = Face()
 
     p₀ = formulation.reference_state.base_pressure
     Nz = grid.Nz
-    bᴺ = ℑzᵃᵃᶠ(i, j, Nz+1, grid, ρ_bᶜᶜᶜ, formulation, args...)
+    b¹ = ℑzᵃᵃᶠ(i, j, 1, grid, ρ_bᶜᶜᶜ, formulation, args...)
 
     # ph⁺ - phᵏ = Δz * b
     # ph⁺ = phᵏ + Δz * b
-    @inbounds ph[i, j, 1] = p₀ - bᴺ * Δzᶜᶜᶠ(i, j, Nz+1, grid)
 
+    @inbounds ph[i, j, 1] = p₀ - b¹ * Δzᶜᶜᶠ(i, j, 1, grid) *0.5
     # Integrate update downwards
     for k in 2:Nz
         bᵏ = ℑzᵃᵃᶠ(i, j, k, grid, ρ_bᶜᶜᶜ, formulation, args...)
-        @inbounds ph[i, j, k] = ph[i, j, k-1] + bᵏ * Δzᶜᶜᶠ(i, j, k, grid)
+        @inbounds ph[i, j, k] = ph[i, j, k-1] - bᵏ * Δzᶜᶜᶜ(i, j, k, grid)
```

Comment:
```suggestion
        @inbounds ph[i, j, k] = ph[i, j, k-1] - bᵏ * Δzᶜᶜᶠ(i, j, k, grid)
```

the face metric is correct because it arises from the pressure derivative

Reply:
The code `ᶜᶜᶜ` refers to the "location of the operator". Therefore `ᶜᶜᶜ` represents the distance between faces, and is located at cell centers. A z-metric at `ᶜᶜᶠ` is located at faces and thus represents the vertical distance between cell centers.

The purpose of this convention is to be able to more easily identify when an operation is correctly reconstructed, eg in correct computations all quantities must have the same code.

Reply:
Yes -- but `Δzᶜᶜᶜ(i, j, 1, grid) / 2` to avoid floating point promotion.